helm/fiaas-skipper/Chart.yaml

```diff
@@ -17,4 +17,4 @@ apiVersion: v1
 description: Skipper controls deployment and updates of FIAAS components
 name: fiaas-skipper
 home: https://github.com/fiaas/skipper
-version: 0.1.0
```

Comment:
Updating the minor version is not needed and will be added by the deploy scripts.

Reply:
Reverted that change.